projects/docker.com/cli/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://github.com/docker/cli/archive/v{{version}}.tar.gz
+  strip-components: 1
+versions:
+  github: docker/cli/tags
+dependencies:
+  darwin:
+    github.com/abiosoft/colima: '*'
```

Comment:
this should be removed pls

there is no mandatory dep

Reply:
👌🏻